blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+---
+title: “Blog- Team Serv0101”
+layout: post
+created_at: Wed Sept 13 2017
+permalink: blog/2017-09-13-final-blog-post-team-Serv0101
```

Comment:
File name and permalink value doesn't match. There is an extra 1 in the file name. Fix it.